docs/cryptodoc/src/05_09_frodokem.rst

```diff
@@ -0,0 +1,298 @@
+.. _pubkey/frodokem:
+
+FrodoKEM
+=================
+
+Botan's FrodoKEM implementation is found in
+:srcref:`src/lib/pubkey/frodokem/` and follows [FrodoKEM-ISO]_.
+The parameter sets shown in the tables below are supported.
```

Comment:
Could it be useful to mention here that eFrodoKEM is a variant prohibiting key reuse?